util/reggen/multi_register.py

```diff
@@ -169,3 +207,39 @@ def _asdict(self) -> Dict[str, object]:
         rd['compact'] = str(self.compact)
 
         return {'multireg': rd}
+
+    def apply_alias(self, alias_reg: object, where: str) -> None:
+        '''
+        Compare all attributes and replace overridable values.
+
+        This ensures both registers are identical up to the overridable
+        attributes like 'name', 'desc', 'resval' and 'tags'.
+        '''
+        if not isinstance(alias_reg, MultiRegister):
+            raise ValueError('alias_reg is not of type MultiRegister')
+
+        # First, validate and update the multireg itself.
```

Comment:
As before, I'm not a fan of setattr/getattr. Probably better to expand out the list of fields explicitly.